src/commands/openYAMLConfigFile.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Uri, window, workspace } from "vscode";
+import { IActionContext } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+import { EnvironmentTreeItem } from "../tree/EnvironmentTreeItem";
+import { StaticWebAppTreeItem } from "../tree/StaticWebAppTreeItem";
+import { openUrl } from "../utils/openUrl";
+
+export async function openYAMLConfigFile(context: IActionContext, node?: StaticWebAppTreeItem | EnvironmentTreeItem): Promise<void> {
+    if (!node) {
+        node = await ext.tree.showTreeItemPicker<EnvironmentTreeItem>(EnvironmentTreeItem.contextValue, context);
+    }
+
+    const defaultHostname: string = node instanceof StaticWebAppTreeItem ? node.data.properties.defaultHostname : node.parent.data.properties.defaultHostname;
+    const ymlFileName: string = `.github/workflows/azure-static-web-apps-${defaultHostname.replace('.azurestaticapps.net', '')}.yml`;
+
+    if (node instanceof EnvironmentTreeItem && node.inWorkspace) {
+        const ymlUri: Uri[] = await workspace.findFiles(ymlFileName);
```

Comment:
I'd use `fse.pathExists` instead of `workspace.findFiles`

Reply:
Any particular reason why?  Performance reasons?

Reply:
Yeah should be more efficient